particles/Tutorial/Javascript/Demo/source/Game.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+const resetMove = {type: 'reset', move: -1};
+const clearMove = {type: 'clear', move: -1};
```

Comment:
When we load particles in WebWorkers, they enter a global scope. Move these definitions inside the `defineParticle` function to keep them from clobbering or being clobbered by other definitions.